### ISSUE TITLE
Bug: ERROR getting IPtables version: exit status 1

### ISSUE THREAD

**Issue by Shadex9** [qdm12/gluetun#2260]
### Is this urgent?

Yes

### Host OS

Synology DSM6.2

### CPU arch

x86_64

### VPN service provider

ProtonVPN

### What are you using to run the container

docker-compose

### What is the version of Gluetun

Running version latest built on 2024-05-02T16:57:20.083Z (commit fb145d6)

### What's the problem 🤔

When attempting to start the container it errors out and auto shuts down. This problem occured this morning after updating. Rollback to V3.38 does make it work again.

### Share your logs (at least 10 lines)

```plain text
INFO [routing] default route found: interface eth0, gateway 172.18.0.1, assigned IP 172.18.0.2 and family v4
INFO [routing] local ethernet link found: eth0
INFO [routing] local ipnet found: 172.18.0.0/16
INFO [firewall] enabling...
INFO [firewall] enabled successfully
NFO [storage] merging by most recent 19425 hardcoded servers and 19425 servers read from /gluetun/servers.json
INFO Alpine version: 3.19.1
INFO OpenVPN 2.5 version: 2.5.8
INFO OpenVPN 2.6 version: 2.6.8
INFO Unbound version: 1.19.3
ERROR getting IPtables version: exit status 1
INFO Shutdown successful
```

### Share your configuration

```yml
gluetun:
    image: qmcgaw/gluetun:latest
    container_name: gluetun
    environment:
      - VPN_SERVICE_PROVIDER=protonvpn
      - OPENVPN_USER=
      - OPENVPN_PASSWORD=
      - SERVER_COUNTRIES=Netherlands
      - SERVER_CITIES=amsterdam
      - TZ=
    volumes:
      - /volume1/docker/gluetun:/gluetun
    ports:
      - 8112:8112
      - 6881:6881
      - 6881:6881/udp
    cap_add:
      - NET_ADMIN
    devices:
      - /dev/net/tun:/dev/net/tun
    restart: always
```

**Comment by CharlesVDP** [comment]
I'm sure this isn't the comment you were hoping or expecting to see, but I just had to let you know that I've been tearing my hair out over the past 24 hours trying to troubleshoot this same error (also using ProtonVPN btw and also on a NAS (asustor), could be relevant) but as I'm new to all things docker so I expected that I was doing something wrong as I've never spun up a gluetun container before!

I'm so happy right now, using v3.38 is indeed working! You've saved the day for me tbh.

**Comment by danwelcome** [comment]
I am having this same issue! Also ruining on a synology nas.

**Comment by qdm12** [comment]
This should be fixed by ef6874fe57500ec322469af4d6781e7e4a9e0719 let me know.
Related to #2256

**Comment by Shadex9** [comment]
I can confirm it is working again. @qdm12 Thanks for the quick fix much appreciated.
